roadmap.md

```diff
@@ -0,0 +1,10 @@
+# Q2 2019
+1. Distirbuted trianing TF (1.x and 2.0)
+1. Distirbuted training for LightGBM
+1. TF-Serving as an option for delpolyment in addition to Seldon model server
+1. Improve documentation and testing
+
+# Q3 and Q4 2019 Big Rocks
+1. Support for distributed training in other frameworks: PyTorch, Spark ML, etc.
+1. Integrations: KF-Serving, KF-Pipelines, Katib
+1. Integration tests for on-prem use case
```

Comment:
Can we add another item for better support for AppendBuilder?

Reply:
done.